### DISCUSSION TITLE
Update to 20221117 doesn't seem to finish

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1093] [Fixed Issues]
@rern 

Just upgraded to 20221117 on RPi 3B+, the update seemed to go OK until the point where the process should end. I get:

/srv/http/settings/features.php
/srv/http/settings/guide.php
/srv/http/settings/networks.php
/srv/http/settings/player.php
/srv/http/settings/relays.php
/srv/http/settings/system.php

 .  Rearrange MPD Configuration...

 .  Done.

But no confirmation dialogue box to end the update process and scan the logs.

Chris

**Comment by rern** [comment]
Sorry. The glitches have been fixed.
It's been a little too long and there're quite a lot to update.
The confirmation dialog will be there for next updates.

**Comment by cYCL157** [comment]
I got some errors in my update, here's the output of the command:

curl -sfLO https://github.com/rern/rAudio-1/raw/main/install.sh
chmod 755 install.sh
./install.sh "r1
Update
20221117"



 .  Update rAudio ...



 .  Get files ...



 .  Install new files ...

/srv/http/assets/css/addons.css
/srv/http/assets/css/colors.css
/srv/http/assets/css/common.css
/srv/http/assets/css/desktop.css
/srv/http/assets/css/equalizer.css
/srv/http/assets/css/keyboard.css
/srv/http/assets/css/main.css
/srv/http/assets/css/relays.css
/srv/http/assets/css/roundslider-1.6.1.min.css
/srv/http/assets/css/selectric.css
/srv/http/assets/css/settings.css
/srv/http/assets/css/simple-keyboard-3.4.139.min.css
/srv/http/assets/data/regdomcodes.json
/srv/http/assets/data/songs_with_trailing
/srv/http/assets/data/system-i2s.json
/srv/http/assets/fonts/Inconsolata.woff2
/srv/http/assets/fonts/Lato-Light.woff2
/srv/http/assets/fonts/Lato-Regular.woff2
/srv/http/assets/fonts/rern.woff2
/srv/http/assets/img/addons/thumbdab.jpg
/srv/http/assets/img/addons/thumbfran.gif
/srv/http/assets/img/addons/thumbgpio.gif
/srv/http/assets/img/addons/thumbplst.jpg
/srv/http/assets/img/addons/thumbrank.png
/srv/http/assets/img/addons/thumbwebr.png
/srv/http/assets/img/camilladsp.svg
/srv/http/assets/img/coverart.svg
/srv/http/assets/img/gpio.svg
/srv/http/assets/img/i2cbackpack.jpg
/srv/http/assets/img/icon.png
/srv/http/assets/img/icon.svg
/srv/http/assets/img/iconcover.svg
/srv/http/assets/img/lcd.jpg
/srv/http/assets/img/lcdchar.jpg
/srv/http/assets/img/mpdoled.jpg
/srv/http/assets/img/powerbutton.svg
/srv/http/assets/img/relays.jpg
/srv/http/assets/img/rotaryencoder.jpg
/srv/http/assets/img/splash.png
/srv/http/assets/img/spotify.gif
/srv/http/assets/img/stopwatch.svg
/srv/http/assets/img/vu.svg
/srv/http/assets/img/vuled.svg
/srv/http/assets/js/addons.js
/srv/http/assets/js/common.js
/srv/http/assets/js/context.js
/srv/http/assets/js/equalizer.js
/srv/http/assets/js/features.js
/srv/http/assets/js/function.js
/srv/http/assets/js/keyboard.js
/srv/http/assets/js/main.js
/srv/http/assets/js/networks.js
/srv/http/assets/js/passive.js
/srv/http/assets/js/player.js
/srv/http/assets/js/plugin/Sortable-1.15.0.min.js
/srv/http/assets/js/plugin/html5kellycolorpicker-1.21.min.js
/srv/http/assets/js/plugin/jquery-3.6.1.min.js
/srv/http/assets/js/plugin/jquery.selectric-1.13.1.min.js
/srv/http/assets/js/plugin/lazysizes-5.3.2.min.js
/srv/http/assets/js/plugin/pica-9.0.1.min.js
/srv/http/assets/js/plugin/pushstream-20211210.min.js
/srv/http/assets/js/plugin/qrcode.min.js
/srv/http/assets/js/plugin/roundslider-1.6.1.min.js
/srv/http/assets/js/plugin/simple-keyboard-3.4.139.min.js
/srv/http/assets/js/relays.js
/srv/http/assets/js/settings.js
/srv/http/assets/js/shortcut.js
/srv/http/assets/js/system.js
/srv/http/bash/addons.sh
/srv/http/bash/albumthumbnail.sh
/srv/http/bash/audiocd.sh
/srv/http/bash/bluealsa-dbus.py
/srv/http/bash/bluetoothbutton.sh
/srv/http/bash/bluetoothcommand.sh
/srv/http/bash/cmd.sh
/srv/http/bash/cmd-coverartsave.sh
/srv/http/bash/cmd-list.sh
/srv/http/bash/cmd-listsort.php
/srv/http/bash/cmd-tageditor.sh
/srv/http/bash/common.sh
/srv/http/bash/dab-scan.sh
/srv/http/bash/dab-start.sh
/srv/http/bash/lcdchar.py
/srv/http/bash/lcdcharconfig.py
/srv/http/bash/lcdcharinit.py
/srv/http/bash/motd.sh
/srv/http/bash/mpdidle.sh
/srv/http/bash/powerbutton.sh
/srv/http/bash/rotaryencoder.sh
/srv/http/bash/scrobble.sh
/srv/http/bash/settings/camilladsp-gain.py
/srv/http/bash/settings/camilladsp-setformat.sh
/srv/http/bash/settings/features.sh
/srv/http/bash/settings/features-data.sh
/srv/http/bash/settings/features-stoptimer.sh
/srv/http/bash/settings/networks.sh
/srv/http/bash/settings/networks-data.sh
/srv/http/bash/settings/networks-scan.sh
/srv/http/bash/settings/player.sh
/srv/http/bash/settings/player-asound.sh
/srv/http/bash/settings/player-conf.sh
/srv/http/bash/settings/player-data.sh
/srv/http/bash/settings/player-devices.sh
/srv/http/bash/settings/player-wm5102.sh
/srv/http/bash/settings/relays.sh
/srv/http/bash/settings/relays-data.sh
/srv/http/bash/settings/relays-timer.sh
/srv/http/bash/settings/system.sh
/srv/http/bash/settings/system-data.sh
/srv/http/bash/settings/system-datareset.sh
/srv/http/bash/shairport.sh
/srv/http/bash/shairport-meta.sh
/srv/http/bash/snapcast.sh
/srv/http/bash/spotifyd.sh
/srv/http/bas

h/startup.sh
/srv/http/bash/status.sh
/srv/http/bash/status-bluetooth.sh
/srv/http/bash/status-coverart.sh
/srv/http/bash/status-coverartonline.sh
/srv/http/bash/status-dab.sh
/srv/http/bash/status-push.sh
/srv/http/bash/status-radio.sh
/srv/http/bash/upmpdcli.sh
/srv/http/bash/vu.sh
/srv/http/bash/xinitrc
/srv/http/camilla.php
/srv/http/cmd.php
/srv/http/common.php
/srv/http/data/mpdconf/conf/autoupdate.conf
/srv/http/data/mpdconf/conf/buffer.conf
/srv/http/data/mpdconf/conf/cdio.conf
/srv/http/data/mpdconf/conf/ffmpeg.conf
/srv/http/data/mpdconf/conf/fifo.conf
/srv/http/data/mpdconf/conf/httpd.conf
/srv/http/data/mpdconf/conf/normalization.conf
/srv/http/data/mpdconf/conf/outputbuffer.conf
/srv/http/data/mpdconf/conf/replaygain.conf
/srv/http/data/mpdconf/conf/snapserver.conf
/srv/http/data/mpdconf/conf/soxr.conf
/srv/http/data/mpdconf/conf/soxr-custom.conf
/srv/http/data/mpdconf/curl.conf
/srv/http/data/mpdconf/mpd.conf
/srv/http/function.php
/srv/http/index.php
/srv/http/login.php
/srv/http/main.php
/srv/http/mpdlibrary.php
/srv/http/mpdplaylist.php
/srv/http/settings.php
/srv/http/settings/addons.php
/srv/http/settings/addons-progress.php
/srv/http/settings/camillagui/backend/views.py
/srv/http/settings/camillagui/build/camillagui.css
/srv/http/settings/camillagui/build/index.html
/srv/http/settings/camillagui/build/static/css/main.css
/srv/http/settings/camillagui/build/static/css/main.css.map
/srv/http/settings/camillagui/build/static/js/main.js
/srv/http/settings/camillagui/build/static/js/main.js.LICENSE.txt
/srv/http/settings/camillagui/build/static/js/main.js.map
/srv/http/settings/features.php
/srv/http/settings/guide.php
/srv/http/settings/networks.php
/srv/http/settings/player.php
/srv/http/settings/relays.php
/srv/http/settings/system.php


 .  Rearrange MPD Configuration...


Terminated




 .  Done.


/srv/http/bash/common.sh: line 49: echo: write error: Broken pipe

/srv/http/bash/status.sh: line 20: echo: write error: Broken pipe
